internal/server/singleprocess/service_job.go

```diff
@@ -160,6 +166,48 @@ func (s *service) queueJobReqToJob(
 		}
 	}
 
+	_, anyTarget := job.TargetRunner.Target.(*pb.Ref_Runner_Any)
+
+	var od *pb.Ref_OndemandRunner
+
+	if anyTarget {
```

Comment:
Personally would compress all the whitespace above here and make this one line:

`if _, anyTarget := job.TargetRunner.Target.(*pb.Ref_Runner_Any); anyTarget {`